server.js

```diff
@@ -59,9 +59,68 @@ function commandRouter({ user_id, user_name, text }, response) {
             firebase.deleteUser(user_id);
             response.end(":'( Goodbye " + user_name + ". It's been real.");
             break;
+        case 'add':
```

Comment:
I assume one doesn't write the # in front of the channel name cause its reserved?

Reply:
you can do either or